pom.xml

```diff
@@ -223,8 +223,8 @@
         <version.gnu.getopt>1.0.13</version.gnu.getopt>
         <version.groovy-all>2.4.7</version.groovy-all>
         <version.httpunit>1.7.2</version.httpunit>
-        <version.io.netty>4.1.9.Final</version.io.netty>
-        <version.io.reactivex.rxjava>2.1.9</version.io.reactivex.rxjava>
+        <version.io.netty>4.1.25.Final</version.io.netty>
```

Comment:
@jmesnil Is this okay for messaging? I would imagine so since it's a micro.

Reply:
@jamezp The feature branch for the artemis upgrade already uses 4.1.24.Final.

Reply:
Perfect thanks! I probably should have just checked :)

Reply:
+1